scvi/inference/inference.py

```diff
@@ -41,28 +41,51 @@ def __init__(
         train_size=0.8,
         test_size=None,
         n_epochs_kl_warmup=400,
+        normalize_loss=None,
```

Comment:
this should be a bool

Reply:
Are we sure we want to fix it to a bool at default value ? So far, scVI only relied on `normalize_loss=True` and all results from AutoZI we now have rely on `normalize_loss=False`. `normalize_loss=None` by default and the processing done for this case helps us treat the two different values for scVI and AutoZI at the same time.

Reply:
can you just clarify in the docstring what the options are?